.github/workflows/deploy.yml

```diff
@@ -0,0 +1,98 @@
+name: Deploy site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+  # Run daily to check for new events
+  schedule:
+    - cron: '0 13 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+# and new content from fetched events
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "lts/*"
+          cache: npm
+          cache-dependency-path: techcity/services/builder/package-lock.json
+
+      - name: Install Node dependencies
+        run: npm --prefix techcity/services/builder ci
+
+      - name: Build CSS file
+        run: npm --prefix techcity/services/builder run css
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@v2
+
+      - name: Set up Python
+        run: uv python install
+
+      - name: Install the project
+        run: uv sync
+
+      - name: Set up the database
+        run: uv run ./manage.py migrate
+
+      - name: Ingest DB
+        run: uv run ./manage.py loaddata db.yaml
+
+      - name: Fetch any new events
+        run: uv run ./manage.py fetch_events
+
+      - name: Write DB back to filesystem
+        run: uv run ./manage.py dumpdata --format=yaml > db.yaml
+
+      - name: Build HTML files
+        run: uv run ./manage.py distill-local out --collectstatic --force
```

Comment:
added this file back,
this bit here is the main change:
it first loads the yaml file into the db, runs "fetch events" command, then writes db back to yaml file

_(I HAVE NOT TESTED THIS YET)_